app/javascript/_utils/isOfficeAdmin.js

```diff
@@ -0,0 +1,17 @@
+export const isOfficeAdmin = account => {
+  return (
+    account !== undefined &&
+    account.admin_office_ids &&
+    !isStateAdmin(account) &&
+    !isCountyOrStateAdmin(account) &&
+    account.roles.includes('Office-admin')
+  );
+};
+
+const isStateAdmin = account => {
+  return account.roles.includes('State-admin');
+};
+
+const isCountyOrStateAdmin = account => {
+  return !isStateAdmin(account) && account.roles.includes('County-admin');
```

Comment:
this code does not agree with the method name.  
```suggestion
  return isStateAdmin(account) || isCountyAdmin(account);
```

Reply:
And it's not covered by a unit test.

Reply:
All branches covered.